.gitignore

```diff
@@ -1,2 +1,3 @@
 .vagrant/
 .www/
+.DS_Store
```

Comment:
won't we need to gitignore every instance of .DS_Store then? such as inside of individual folders?